api/install.sh

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash
 
-apt update
-apt upgrade -y
-apt install -y golang
-
 rm -rf api
 git clone https://github.com/tune-bot/api.git
 
-mkdir -p bin
```

Comment:
was this line not necessary?

Reply:
Oh I see, you moved it. I intended each script to be self contained and be able to run independently.

Reply:
Cool, I can revert those changes